olive/systems/python_environment/available_eps.py

```diff
@@ -0,0 +1,17 @@
+import argparse
+import pickle
+from pathlib import Path
+
+import onnxruntime as ort
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--output_path", type=str, required=True)
+    args = parser.parse_args()
+
+    # get available execution providers
+    available_eps = ort.get_available_providers()
```

Comment:
What will happed if the setup is not ready for available_providers? In a few cases, the `ort.get_available_providers()` will return tensorrt, cuda and cpu eps, but the local env tensorrt is not ready. When it comes to inferences settings creation, the program crashes.

Reply:
I have updated the logic to check if EP is valid for the model like it is done in `ONNXModel`. Does that solve this issue?